src/components/PhotoAnalysis/rekognitionUtils.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { BoundingBox } from "@aws-amplify/predictions"
+import CSS from "csstype"
+
+import { LabelType } from "./PhotoAnalysis"
+import { Label } from "../Label/Label"
+
+export const labelImage = (labelData: LabelType[], imageSrc: string) => {
+  const img = new Image()
+  img.src = imageSrc
+  const height = img.height
+  const width = img.width
+  return labelData.map(label => {
+    if (label.boundingBoxes.length > 0) {
+      let style
+      label.boundingBoxes.forEach((boundingBox: BoundingBox) => {
+        style = defineBoundingBox(boundingBox, height, width)
+      })
+      return <Label value={label.name} style={style} key={label.name} />
```

Comment:
This is almost there I think, but there's something missing because the labels still display below the image